tools/harness-thci/OpenThread.py

```diff
@@ -1511,14 +1511,18 @@ def reset(self):
 
         start_time = time.time()
         while time.time() < start_time + timeout:
-            time.sleep(0.3)
+            time.sleep(0.5)
```

Comment:
Is the delay change significant?

Reply:
We were finding that with some of our devices, the reset did not complete prior to the execution of the state command.  The smaller value is likely fine over SPI but UART devices can be a little slower.